### ISSUE TITLE
[BUG] Remote end closed connection without response

### ISSUE THREAD

**Issue by trollollol123** [jurialmunkey/plugin.video.themoviedb.helper#1292]
### Describe the bug

info <general>: [plugin.video.themoviedb.helper]
ConnectionError: Not connected. Check network settings.
 ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
Suppressing retries.

### Steps To Reproduce

Nothing, just watch something.

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I have updated TMDbHelper to the most recent version.
- [X] I have described the bug and included steps to reproduce the issue
- [X] I have uploaded my debug log.

**Comment by jurialmunkey** [comment]
No log uploaded.

**Comment by smokyy12** [comment]
I believe i have the same problem since a few weaks, error message is: 

[plugin.video.themoviedb.helper ConnectionError:  ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')) Suppressing retries for 15 seconds

I get i on a regular basis while watching something and also in the menu. I also read some reddit posts of people who have the same error and someone posted it on the arctic fuse 2 thread in the kodi forum, so i believe its a common error atm. 

Heres a log file: 
[kodi.log](https://github.com/user-attachments/files/18168944/kodi.log)

**Comment by jurialmunkey** [comment]
And what problem is it causing?

**Comment by trollollol123** [comment]
I haven't noticed any "problems" myself yet, other than the annoying message popping up while watching something and in the menu.

It says something like: ConnectionError TraktAPI
And I get this even tho I logged in/out of trakt and authorized/deauthorized my device.

**Comment by jurialmunkey** [comment]
You shouldn't be getting a notification for this error with latest version of script.module.jurialmunkey which is why I asked for a log. The error should only be logged not notified.

**Comment by trollollol123** [comment]
> You shouldn't be getting a notification for this error with latest version of script.module.jurialmunkey which is why I asked for a log. The error should only be logged not notified.

Yea sorry I just havent had time to get a clean log yet. But the poster above me posted his log and he stated "I get i on a regular basis while watching something and also in the menu.", so he seems to have exactly my problem.

**Comment by jurialmunkey** [comment]
> > You shouldn't be getting a notification for this error with latest version of script.module.jurialmunkey which is why I asked for a log. The error should only be logged not notified.
> 
> Yea sorry I just havent had time to get a clean log yet. But the poster above me posted his log and he stated "I get i on a regular basis while watching something and also in the menu.", so he seems to have exactly my problem.

Yes, but are you using the latest version of the module? ~Because you shouldn't get the notification now. It should only be logged, not notified.~

If the only problem is the notification and it is not causing other issues then that has already been addressed. ~However, if you are using latest version and still getting the notification then that indicates I need to fix something in the notification code.~

EDIT: Realised that I didn't push the notification suppression code before I last update the repo, so you would still be getting notifications. Will address that in next update.

**Comment by jurialmunkey** [comment]
> I believe i have the same problem since a few weaks, error message is:
> 
> [plugin.video.themoviedb.helper ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')) Suppressing retries for 15 seconds
> 
> I get i on a regular basis while watching something and also in the menu. I also read some reddit posts of people who have the same error and someone posted it on the arctic fuse 2 thread in the kodi forum, so i believe its a common error atm.
> 
> Heres a log file: [kodi.log](https://github.com/user-attachments/files/18168944/kodi.log)

@smokyy12 - Is this causing an issue though? Or is it just a case of the notifications being annoying?

**Comment by trollollol123** [comment]
> 
> If the only problem is the notification and it is not causing other issues then that has already been addressed. However, if you are using latest version and still getting the notification then that indicates I need to fix something in the notification code.

I am using the latest ([v5.4.18](https://github.com/jurialmunkey/plugin.video.themoviedb.helper/releases/tag/v5.4.18)). BUT there is an option in settings under "other"-> "connection error notifications", this was CHECKED by default for me. I think maybe unchecking this might have removed the errors for me. But I only found this setting a short while ago so I haven't tested it for that long, so I can't say 100% that it removed the messages yet.

**Comment by jurialmunkey** [comment]
> I am using the latest ([v5.4.18](https://github.com/jurialmunkey/plugin.video.themoviedb.helper/releases/tag/v5.4.18)).

That's not `script.module.jurialmunkey` ....

>   BUT there is an option in settings under "other"-> "connection error notifications" ... I think maybe unchecking this might have removed the errors for me.

Yes that is what that setting does. It suppresses all connection error notifications.

**Comment by trollollol123** [comment]
> > I am using the latest ([v5.4.18](https://github.com/jurialmunkey/plugin.video.themoviedb.helper/releases/tag/v5.4.18)).
> 
> That's not `script.module.jurialmunkey` ....

Sorry, I'm not an advanced user, so I'm not exactly sure how to update that module. But according to my current log, it says:

info <general>: CAddonMgr::FindAddons: script.module.jurialmunkey v0.2.6 installed

Edit: but like I said, that option for errors was checked by default for me. So maybe the issues is indeed fixed like you said.

**Comment by smokyy12** [comment]
@jurialmunkey no i did not notice any issues, trakt is working fine. its just the error message. but i did try to uncheck the „show error notification“ setting and still got the error messages displayed.
